src/qibolab/platform.py

```diff
@@ -1,3 +1,109 @@
+from qibo.config import raise_error
+
+from qibolab.designs.basic import BasicInstrumentDesign
+from qibolab.designs.mixer import MixerInstrumentDesign
+from qibolab.platforms.platform import DesignPlatform
+
+
+def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=False):
+    """Create platform using Quantum Machines (QM) OPXs and Rohde Schwarz local oscillators.
+
+    IPs and other instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+
+    Args:
+        runcard (str): Path to the runcard file.
+        simulation_duration (int): Duration for the simulation in ns.
+            If given the compiler simulator will be used instead of the actual hardware.
+            Default is ``None`` which falls back to the hardware.
+        address (str): Address and port for the QM OPX cluster.
+            If ``None`` it will attempt to connect to TII instruments.
+        cloud (bool): See :class:`qibolab.instruments.qmsim.QMSim` for details.
+            Relevant only when ``simulation_duration`` is given.
+    """
+    # Instantiate QM OPX instruments
+    if simulation_duration is None:
+        from qibolab.instruments.qm import QMOPX
+        from qibolab.instruments.rohde_schwarz import SGS100A
```

Comment:
This can also be just a comment to the original PR (i.e. #259, not directly this one), but: why do you need to import here instead of at top-level?

Reply:
Following my response [here](https://github.com/qiboteam/qibolab/pull/259#discussion_r1085660913), the idea is to allow using part of the code without having to install all dependencies. In this example, one may want to use the simulation mode without installing the libraries needed to connect to LOs (py-visa, etc.). Another example is if one wants to use qblox (a different platform) they may don't want to install QM libraries. Let me know if you think that's a valid point.

Reply:
> Following my response [here](https://github.com/qiboteam/qibolab/pull/259#discussion_r1085660913), the idea is to allow using part of the code

Sorry, my bad, I missed the notification about the other reply. Then, it is fine as you did.